rbac/common/sawtooth/batcher.py

```diff
@@ -23,9 +23,14 @@
 from rbac.app.config import BATCHER_KEY_PAIR
 from rbac.common.sawtooth import rbac_payload
 
+
 LOGGER = logging.getLogger(__name__)
 
 
+def get_message_type_name(message_type):
+    return rbac_payload.get_message_type_name(message_type=message_type)
```

Comment:
Docstring please

Reply:
It's not worth the time to rebase #536 to add docstrings to #532.
Docstrings will be added in a followup PR.

Reply:
@adamgering, you have blocked PRs in the past week over:
  - whitespace change to a file, performed automatically by Black  
  - removal of the 3-4th layer of env variable redundancy having no effect on the master branch

Those were apparently worth the time of others.

We have agreed to docstring our code.

Reply:
I asked for PRs not to change files that have nothing to do with the PR.
I asked for a PR not to remove a function that was added for a required purpose.
Blocking PRs as a retaliation seems pretty and unprofessional and puts the deliverables at risk. Docstring are not yet required by our lint standards.